front/components/assistant/conversation/UserMessage.tsx

```diff
@@ -32,6 +40,20 @@ export function UserMessage({
     agentsGetView: { conversationId },
   });
 
+  const [editing, setEditing] = useState(false);
```

Comment:
Is that resilient to the user starting a new user message while this is true?

Reply:
This is specific to one message - new message will have a new state (false)